include/avif/avif.h

```diff
@@ -85,8 +85,11 @@ extern "C" {
     ((AVIF_VERSION_MAJOR * 1000000) + (AVIF_VERSION_MINOR * 10000) + (AVIF_VERSION_PATCH * 100) + AVIF_VERSION_DEVEL)
 
 typedef int avifBool;
-#define AVIF_TRUE 1
-#define AVIF_FALSE 0
+enum avifTrueFalse
```

Comment:
Do we still get the benefit of the `-Wenum-conversion` warning if this enum is unnamed?`

```
enum
{
    AVIF_FALSE = 0,
    AVIF_TRUE = 1
};
```

Reply:
actually yes. We can name it though